templates/team-listing.html

```diff
@@ -1,18 +1,46 @@
 {% set unprivileged = not(participant == user.participant or user.ADMIN) %}
 {% set approved_open_teams = participant.get_teams(unprivileged, unprivileged) %}
-
-{% if (user.ADMIN or user.participant == participant) and approved_open_teams %}
+{% if (user.ADMIN or user.participant == participant) or approved_open_teams %}
```

Comment:
Currently we show *all* teams to admins and owners, whether or not they're approved or open. We only constrain for non-admin/owner. I think we want to preserve that behavior.